packages/models-library/src/models_library/docker.py

```diff
@@ -37,6 +38,8 @@ class SimcoreServiceDockerLabelKeys(BaseModel):
     user_id: UserID = Field(..., alias="user_id")
     project_id: ProjectID = Field(..., alias="study_id")
     node_id: NodeID = Field(..., alias="uuid")
+    product_name: Optional[ProductName] = None
```

Comment:
why Optional?

Reply:
ok so my next question is:
why not making it directly non optional and using a validator if the value does not exist such as
```
def migrate_from_old_version(v):
```
just so you do not need an issue. we do not forget it. and it works also for already running sidecars.